src/Array.ts

```diff
@@ -832,6 +1017,29 @@ export const uniq = <A>(S: Setoid<A>): ((as: Array<A>) => Array<A>) => {
 /**
  * Sort the elements of an array in increasing order, where elements are compared using first `ords[0]`, then `ords[1]`,
  * etc...
+ *
+ * @example
+ * import { contramap, ordString, ordNumber } from 'fp-ts/lib/Ord'
+ *
+ * interface Person {
+ *   name: string
+ *   age: number
+ * }
+ * const byName = contramap((p: Person) => p.name, ordString)
+ * const byAge = contramapOrd((p: Person) => p.age, ordNumber)
```

Comment:
should be just `contramap`
```suggestion
 * const byAge = contramap((p: Person) => p.age, ordNumber)
```